src/bsoncxx/include/bsoncxx/v_noabi/bsoncxx/config/util.hpp

```diff
@@ -0,0 +1,125 @@
+// clang-format off
+/**
+ * @brief Convert the given macro argument to a string literal, after macro expansion
+ */
+#define bsoncxx_stringify(...) bsoncxx_stringify_impl(__VA_ARGS__)
+#define bsoncxx_stringify_impl(...) #__VA_ARGS__
+
+/**
+ * @brief Token-paste two macro arguments, after macro expansion
+ */
+#define bsoncxx_concat(A, ...) bsoncxx_concat_impl(A, __VA_ARGS__)
+#define bsoncxx_concat_impl(A, ...) A##__VA_ARGS__
+
+/**
+ * @brief Expands to a _Pragma() preprocessor directive, after macro expansion
```

Comment:
Unless we intend to support these macros as part of our public API, recommend either demoting these documentation comments to plain `//` (non-Doxygen) or adding `@internal`+`@endinternal` so they do not show up in the API reference documentation pages.